src/Questions/Input/index.js

```diff
@@ -32,7 +33,24 @@ const QuestionInput = ({ question, useForm, component }) => {
             }
       }
     >
-      {question.label && <Label>{question.label}</Label>}
+      <div sx={{ display: 'flex' }}>
+        {question.label && <Label>{question.label}</Label>}
+        {question.icon && (
+          <div data-event='click focus' data-tip='custom show'>
+            <Logo
+              className='App-logo'
+              aria-label='logo'
+              sx={{ width: '23px', height: '17px' }}
+              fill={question.icon.fill}
+            />
+            {question.icon.tooltipText && (
+              <ReactTooltip globalEventOff='click' effect='solid'>
```

Comment:
As we spoke this should be its own component 😄 with the opening params to be able to config externally